_codux/boards/avatar/avatar.board.tsx

```diff
@@ -0,0 +1,23 @@
+
+export default createBoard({
+    name: 'Avatar',
+    Board: () => {
+        return (
+            <div className={styles.container}>
+                <Avatar imageSrc={undefined} altText="unknown user avatar" />
+
+                <Avatar imageSrc={catImage} altText="Codux" />
```

Comment:
Is it Codux though? Just use an empty. Why do you need an alt on this thing it's not an illustration.